examples/cstg/html/index.html

```diff
@@ -72,33 +93,75 @@
           $(document).ready(onDocumentReady);
         }
       });
+
+      window.__euid = window.__euid || {};
+      window.__euid.callbacks = window.__euid.callbacks || [];
+
+      window.__euid.callbacks.push(onEuidIdentityUpdated);
+      window.__euid.callbacks.push((eventType, payload) => {
+        if (eventType === 'SdkLoaded') {
+          window.__euid.init({
+            baseUrl: '{{ UID2_BASE_URL }}',
+          });
+        }
+      });
     </script>
   </head>
   <body>
-    <h1>Server-Side Integration Example, UID2 JavaScript SDK</h1>
+    <h1>CSTG Integration Example, UID2 JavaScript SDK</h1>
```

Comment:
Is it worth adding "EUID" to the title here? Or even removing "UID2" since it is specified in the table below?

Reply:
I'll update.